org.jacoco.build/pom.xml

```
@@ -740,7 +740,7 @@
             </Export-Package>
             <Import-Package>
               org.jacoco.*;version="${range;[===,==+);${Bundle-Version}}",
-              org.objectweb.asm.*;version="${range;[===,=+);${asm.version}}"
+              org.objectweb.asm.*;version="[9.0.0,10.0)"
```

Seems like an artifact of trial with 9.0 BETA, let's keep substitution from property.
```suggestion
              org.objectweb.asm.*;version="${range;[===,=+);${asm.version}}"
```

---

Correct, this was not supposed to be part of this commit. Fixed.